src/templates/CheckOut/__tests__/checkout.spec.js

```diff
@@ -3,7 +3,20 @@ import { expect } from 'chai';
 import { shallow, render } from 'enzyme';
 
 import CheckOut from 'templates/CheckOut';
-// import styles from 'templates/CheckOut/checkout.scss';
+
+/* eslint-disable */
+jest.mock('atoms/Icon', () => {
```

Comment:
Does this work? If so, I'm glad you discovered this, might be better alternative to 'rewire'